scripts/data/coverage/coverage_merger.rb

```diff
@@ -72,6 +142,9 @@
   parser.on("-c", "--coverageRoot DIR", "Path to the coverage directory.") do |v|
     options[:coverage_root] = v
   end
+  parser.on("-v", "--verbose", "Turn on verbose output.") do 
```

Comment:
:+1:

Reply:
Want to include -v in the calling of this script from CFME tooling?

Reply:
No, that's why I added it.   A lot of that verbose output is way overkill, unless you are changing the code.